stingray/bispectrum.py

```diff
@@ -78,6 +79,23 @@ def compute_bispectrum(self, lc):
         bispec = scipy.fftpack.fftshift(bispec) / self.k
         return bispec
 
+    def plot(self):
```

Comment:
We should maintain consistency in the name of drawing methods of various classes. PR #74 adds the `draw` and `savefig` methods.